src/improvements/tasks/sep/011-opcm-upgrade-prestate-v300-base/VALIDATION.md

```diff
@@ -0,0 +1,163 @@
+- **Key:**          `0x0000000000000000000000000000000000000000000000000000000000000005`
+  - **Decoded Kind:**      `uint256`
+  - **Before:** `21`
+  - **After:** `22`
+  - **Summary:** Nonce update
+  - **Detail:** Nonce update for the parent multisig. You can verify manually with the following:
+    - Before: `cast --to-dec 0x15` = 21
+    - After: `cast --to-dec 0x16` = 22
+
+If signer is on Child Safe 1: `0x6AF0674791925f767060Dd52f7fB20984E8639d8`:
+
+- **Key:**      `0x791fae0bcf3dd62c15434661e854630af982fdfefb63ed5ff80632964121d8ac`
+  - **Before:** `0x0000000000000000000000000000000000000000000000000000000000000000`
+  - **After:** `0x0000000000000000000000000000000000000000000000000000000000000001`
+  - **Summary:**  `approveHash(bytes32)` called on ProxyAdminOwner by child multisig.
+  - **Detail:** **THIS WAS CAREFULLY VERIFIED BY RUNBOOK REVIEWERS AND NEED NOT BE CHECKED BY SIGNERS.** As part of the Tenderly simulation, we want to illustrate the <i>approveHash</i> invocation. This step isn't shown in the local simulation because the parent multisig is invoked directly, bypassing the <i>approveHash</i> calls. This slot change reflects an update to the approvedHashes mapping.
+    Specifically, this simulation was ran as the nested safe ``. To verify the slot yourself, run:
+    - `res=$(cast index address 0x6AF0674791925f767060Dd52f7fB20984E8639d8 8)`
+    - `cast index bytes32 0xf7f42d9617b2d58b07ebbba60ac231142f29f643c82bef5752250bcaa0ef9b34 $res`
+    - Please note: the `0xf7f42d9617b2d58b07ebbba60ac231142f29f643c82bef5752250bcaa0ef9b34` value is taken from the Tenderly simulation and this is the transaction hash of the `approveHash` call.
+
+OR if signer is on Child Safe 2: `0x646132A1667ca7aD00d36616AFBA1A28116C770A`:
+
+- **Key:**      `0x8d756791beca7e82f4f71d8867bfa1c3217415c53f415e97b015e82ea7fd001c`
+  - **Before:** `0x0000000000000000000000000000000000000000000000000000000000000000`
+  - **After:** `0x0000000000000000000000000000000000000000000000000000000000000001`
+  - **Summary:**  `approveHash(bytes32)` called on ProxyAdminOwner by child multisig.
+  - **Detail:** **THIS WAS CAREFULLY VERIFIED BY RUNBOOK REVIEWERS AND NEED NOT BE CHECKED BY SIGNERS.** As part of the Tenderly simulation, we want to illustrate the <i>approveHash</i> invocation. This step isn't shown in the local simulation because the parent multisig is invoked directly, bypassing the <i>approveHash</i> calls. This slot change reflects an update to the approvedHashes mapping.
+    Specifically, this simulation was ran as the nested safe ``. To verify the slot yourself, run:
+    - `res=$(cast index address 0x646132A1667ca7aD00d36616AFBA1A28116C770A 8)`
+    - `cast index bytes32 0xf7f42d9617b2d58b07ebbba60ac231142f29f643c82bef5752250bcaa0ef9b34 $res`
+    - Please note: the `0xf7f42d9617b2d58b07ebbba60ac231142f29f643c82bef5752250bcaa0ef9b34` value is taken from the Tenderly simulation and this is the transaction hash of the `approveHash` call.
+
+  ---
```

Comment:
I see this state diff in here:
![CleanShot 2025-04-11 at 17 45 38](https://github.com/user-attachments/assets/2a602cf5-52a5-47f2-bfd0-1deb71b13c30)

Reply:
And when signing for child-safe-2 I see this one:
![CleanShot 2025-04-11 at 17 46 26](https://github.com/user-attachments/assets/ae35a52b-7537-42a2-9370-ca5161487388)